pkg/scheduler/algorithm/predicates/max_attachable_volume_predicate_test.go

```diff
@@ -991,12 +989,11 @@ func getNodeWithPodAndVolumeLimits(limitSource string, pods []*v1.Pod, limit int
 	case "csinode-with-no-limit":
 		addLimitToCSINode(false)
 	case "no-csi-driver":
-		csiNode := createCSINode()
-		nodeInfo.SetCSINode(csiNode)
-	default:
```

Comment:
Leave an empty default here is better IMO

Reply:
Added back